internal/cli/sqlfmt.go

```diff
@@ -66,10 +66,11 @@ var sqlfmtCtx struct {
 // NewSqlfmtCmd create and returns the sqlfmt command
 func NewSqlfmtCmd() *cobra.Command {
 	sqlfmtCmd := &cobra.Command{
-		Use:   "sqlfmt",
-		Short: "format SQL statements",
-		Long:  "Formats SQL statements from stdin to line length n.",
-		RunE:  runSQLFmt,
+		Use:     "sqlfmt",
+		Version: "1.5.1",
```

Comment:
Thank you for your contribution @JMavs !!!

Can we define the version in a const or get the value present in the `Makefile` somehow? Its 100% likely that I will forget to update the value in two places in the next release and thus, give incorrect data to the user.